batch/batch/worker/worker.py

```diff
@@ -1193,9 +1193,10 @@ async def container_config(self):
                 'resources': {
                     "devices": [{"allow": False, "access": "rwm"}],
                     'cpu': {'shares': weight},
-                    'memory': {
-                        'limit': self.memory_in_bytes,
-                        'reservation': self.memory_in_bytes,
+                    'unified': {  # https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md
+                        'memory.max': str(int(0.95 * self.memory_in_bytes)),
```

Comment:
this takes too much memory away. Something like 0.99 seems reasonable. On a 16-core standard worker VM that keeps 600MB for the worker process. 288MB on high-cpu and 1200MB on highmem.